imap_processing/codice/cdf_attrs.py

```diff
@@ -82,6 +82,15 @@
     instrument_base=codice_base,
 )
 
+l1a_lo_sw_angular_counts_attrs = GlobalDataLevelAttrs(
+    data_type="L1A_SCI->Level-1A Science Data",
```

Comment:
```suggestion
    data_type="L1A_Lo-SW-Angular-Counts->Level-1A Lo Sunward Angular Counts Data",
```

Reply:
If I remember correctly, this `data_type` is used to build filename convention that SPDF and SKTEditor use. That’s why I made suggestion to match the format it expects, `<datalevel>_<descriptor>`. `Logical_source` is used by `write_cdf` function to build our filename we use to write to.